app/controllers/posts_controller.rb

```diff
@@ -11,5 +12,23 @@ def show
     @user = User.find(user_id)
     @post = Post.find(post_id)
     @comments = Comment.where(post_id:)
+    @recent_user = current_user
+  end
+
+  def new
+    @post = Post.new
+  end
+
+  def create
+    user = current_user
+    puts user
```

Comment:
[REQUIRED] Kindly remove this. since we do not need to display anything in the console